src/fullyautomatednutcracker/cogs/fun.py

```diff
@@ -18,6 +18,14 @@ def __init__(self, bot):
         self.bot = bot
         username, password, client_secret, client_id = load_reddit_creds()
         self.bot.reddit = apraw.Reddit(username=username, password=password, client_secret=client_secret, client_id=client_id)
+        self.submissions = []
+
+    def get_submission(self, subreddit):
+        if len(self.submissions) == 0:
+            async for submission in subreddit.hot(limit=25):
+                if submission.pinned is False:
+                    self.submissions.append(submission)
+        return random.choice(self.submissions)
```

Comment:
should you remove the submission from the list after you get a random one?

Reply:
Fuck thats what I forgot lmao